Assets/MixedRealityToolkit-SDK/Features/Diagnostics/System/MixedRealityDiagnosticsManager.cs

```diff
@@ -17,12 +17,8 @@ public class MixedRealityDiagnosticsManager : BaseEventSystem, IMixedRealityDiag
 
         public override void Initialize()
         {
-            base.Initialize();
-            InitializeInternal();
-        }
+            if (!Application.isPlaying) { return; }
```

Comment:
Please revert changes to the Diagnostics system as they are not in the scope of a Focus Provider change.  Please resubmit as a separate PR.

Reply:
Already planning on it.